tests/defender/system/aws_iot_tests_defender_system.c

```diff
@@ -67,10 +68,11 @@
 #define NO_EVENT                             10000
 
 /* Empty callback structure passed to startInfo. */
-static const AwsIotDefenderCallback_t _EMPTY_CALLBACK = { .function = NULL, .param1 = NULL };
+static const AwsIotDefenderCallback_t _EMPTY_CALLBACK = { .function = NULL, .pCallbackContext = NULL };
```

Comment:
The style guide mentions about use of capitals only for `#define`s and **enumerations**. (I had recently given a proposal of updating the style rule to do the same for `const` objects as well, however, that was turned down)

Reply:
That's correct. The earlier style guide had '_' in the beginning, so this define carried over from earlier code and therefore you are seeing styling guide mismatch.